profiles/__init__.py

```diff
@@ -1,11 +1,12 @@
 import os
-
```

Comment:
This newline serves to separate python library imports from third party imports. It helps delineate import ordering visually.

Reply:
Fixed